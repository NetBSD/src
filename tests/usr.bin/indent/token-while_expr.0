/* $NetBSD: token-while_expr.0,v 1.2 2021/09/25 13:26:07 rillig Exp $ */
/* $FreeBSD$ */

/*
 * Tests for the keyword 'while', followed by a parenthesized expression.
 */

int main(int argc,char**argv){int o;while((o=getopt(argc,argv,"x:"))!=-1)
switch(o){case'x':do{o++;}while(o<5);break;default:usage();}return 0;}
