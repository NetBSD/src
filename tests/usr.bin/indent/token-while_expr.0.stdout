/* $NetBSD: token-while_expr.0.stdout,v 1.2 2021/09/25 13:26:07 rillig Exp $ */
/* $FreeBSD$ */

/*
 * Tests for the keyword 'while', followed by a parenthesized expression.
 */

int
main(int argc, char **argv)
{
	int		o;
	while ((o = getopt(argc, argv, "x:")) != -1)
		switch (o) {
		case 'x':
			do {
				o++;
			} while (o < 5);
			break;
		default:
			usage();
/* $ XXX: The 'return' should be in a separate line. */
		} return 0;
}
